greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
We don't impose an 80 character limit on python in mozilla central. It's a subjective matter, but I am strongly of the opinion that in certain cases in python, lines longer than 80 characters are more readable than what is necessary to break them up. I would prefer to leave this particular matter to individual contributors and reviewers.